README.md

```diff
@@ -4,17 +4,16 @@ This repo contains the MFEM website [MkDocs](http://www.mkdocs.org/) sources.
 
 To make changes to the website you will need an install of Python 3 with the following libraries:
 
-- use the latest versions of MkDocs, Markdown, PyYAML and mkdocs-exclude-search:
-  * `pip install --upgrade --user mkdocs`
-  * `pip install --upgrade --user Markdown`
+- use MkDocs v1.0.4 with Markdown v3.0 and the latest PyYAML and mkdocs-exclude-search:
+  * `pip install --upgrade --user mkdocs==1.0.4`
```

Comment:
Unfortunately this version breaks the search functionality.  I'll see if I can help get this working int he newer version.

Reply:
Hmm.. I specifically tried it and it worked for me. Can you do `pip show` on your versions?

Reply:
mkdocs = 1.0.4
markdown = 3.1.1
pyyaml = 5.4.1
mkdocs-exclude-search = 0.5.0

python = 2.7.15rc1

Reply:
Here are mine:

```
> pip show mkdocs Markdown PyYAML mkdocs-exclude-search | grep Version

Version: 1.0.4
Version: 3.0
Version: 5.4.1
Version: 0.5.2
```

Can you maybe try downgrading Markdown?

Reply:
That helped, but it is crashing on something in mkdocs-exclude-search.  For some reason I can't get 0.5.2 of that package.  Maybe I have an older version of python installed here?  What is your python version?

Reply:
I have multiple 😞 , but I think here I'm using 3.9

Reply:
I belive if you do a 'mkdocs --version' it will tell you which python you are running.

Reply:
```
> mkdocs --version
mkdocs, version 1.0.4 from /Library/Python/3.9/lib/python/site-packages/mkdocs (Python 3.9)
```

Reply:
Interesting, when I install with mkdocs==1.0.4 using my python 3 it gives me mkdocs version 1.2

Reply:
I'm going to try upgrading my python3.

Reply:
Ok that fixed it.  I think we should recommend at least python 3.6.3

Reply:
Sorry for all the trouble @acfisher, hopefully mkdocs will be improved and we will be able to use the latest release in the future.